src/palace.pyx

```diff
@@ -2382,7 +2382,7 @@ cdef class ReverbEffect(BaseEffect):
 
     @property
     def decay_hf_limit(self) -> bool:
-        """High frequency decay limit, either TRUE or FALSE."""
+        """High frequency decay limit."""
         return True if self.properties.decay_hf_limit else False
```

Comment:
Ugh missed this til now, what an abomination.  Please change it to `bool(self.properties.decay_hf_limit)`.